eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -0,0 +1,128 @@
+from collections import Counter
+from itertools import chain
+
+
+class DbtModelParser():
+
+    def __init__(self, model, dbt_model_prefix):
+        self.model = model
+        self.eppo_entities = []
+        self.eppo_timestamp = []
+        self.eppo_facts = []
+        self.eppo_properties = []
+        self.dbt_model_prefix = dbt_model_prefix
+    
+    def parse_columns(self):
+
+        for column in self.model.get('columns'):
+
+            tags = column.get('tags')
+
+            if tags:
+                column_entities = [t for t in tags if 'eppo_entity:' in t]
+
+                if column_entities:
+                    for column_entity in column_entities:
+                        column_entity_split = column_entity.split(':')
+                        if len(column_entity_split) != 2:
```

Comment:
The `2` here is a magic number. For cases like this, I think a good simple change is to change

```
if len(column_entity_split) != 2:
```

to 
```
EXPECTED_SPLIT_SIZE=2
if len(column_entity_split) != EXPECTED_SPLIT_SIZE:
```
It's a small difference, but the second case explains what the 2 is, making the code read more like prose. Also a good example of self-documenting code (the code explains itself without using comments)